### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewOutput_ScreenReader: Jaws does not read the "role" of the combobox and instruction to open it.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52195]
Environment Details:
VSCode Version : 1.24.0
OS Version : Win10

Additional Details:
MAS Violated: MAS1.3.1
ScreenReader Version: Jaws 18.0.5038

Repro Steps:
Launch VS Code while NVDA is ON.
Go to View menu and Select Output.
Tab till the combobox.

Actual:
Jaws does not read the role of the combobox and its instruction to open the dropdown,it only reads "Output Actions toolbar read only" and its value selected.

Expected:
Jaws should read the role of the control and instruction to open it with its value selected.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not get the proper information about the control .

MAS Reference
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference:
[A11y VSCode Output Jaws missing role.pptx](https://github.com/Microsoft/vscode/files/2111162/A11y.VSCode.Output.Jaws.missing.role.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by auchenberg** [comment]
Closing this issue as it's only observed in JAWS, and this isn't strictly needed to meet MAS C compliance.
